test/test_python_dispatch.py

```diff
@@ -565,9 +565,11 @@ def test_subclass_creation(self):
         class Foo(torch.Tensor):
             pass
 
+
+        a = torch.Tensor._make_subclass(Foo, LoggingTensor(torch.rand(2)))
+        self.assertIsInstance(a, Foo)
```

Comment:
This now works because we aren't attaching a python object to logging tensor during detach...not certain how we feel about this

Reply:
This doesn't seem good, we're silently getting rid of how the object was previously a LoggingTensor